src/storage/index/IndexPolicyMaker.cpp

```diff
@@ -98,30 +109,27 @@ cpp2::ErrorCode IndexPolicyMaker::traversalExpression(const Expression *expr) {
             auto* rExpr = dynamic_cast<const RelationalExpression*>(expr);
             auto* left = rExpr->left();
             auto* right = rExpr->right();
-            if (left->kind() == nebula::Expression::kAliasProp
-                && right->kind() == nebula::Expression::kPrimary) {
+            if (left->kind() == nebula::Expression::kAliasProp) {
                 auto* aExpr = dynamic_cast<const AliasPropertyExpression*>(left);
                 prop = *aExpr->prop();
-                auto* pExpr = dynamic_cast<const PrimaryExpression*>(right);
-                auto value = pExpr->eval(getters);
+                auto value = right->eval(getters);
                 if (!value.ok()) {
+                    VLOG(1) << "Can't evaluate the expression " << right->toString();
                     return cpp2::ErrorCode::E_INVALID_FILTER;
                 }
                 v = value.value();
-            } else if (left->kind() == nebula::Expression::kPrimary
-                       && right->kind() == nebula::Expression::kAliasProp) {
-                auto* pExpr = dynamic_cast<const PrimaryExpression*>(left);
-                auto value = pExpr->eval(getters);
+            } else if (right->kind() == nebula::Expression::kAliasProp) {
+                auto value = left->eval(getters);
                 if (!value.ok()) {
+                    VLOG(1) << "Can't evaluate the expression " << left->toString();
                     return cpp2::ErrorCode::E_INVALID_FILTER;
                 }
                 v = value.value();
                 auto* aExpr = dynamic_cast<const AliasPropertyExpression*>(right);
                 prop = *aExpr->prop();
             } else {
-                traversalExpression(left);
-                traversalExpression(right);
-                break;
+                optimizedPolicy_ = false;
```

Comment:
Why exit?

Reply:
Index (col1, col2)
For the expression "co1 > 1 and col1 < 10 and col2 > 5",   the index can't be used?

Reply:
This index can be used. This is a standard logical expression. 
for example in my test :
```
CREATE EDGE lookup_edge_1(col1 int, col2 int, col3 int);
CREATE EDGE INDEX e_index_1 ON lookup_edge_1(col1, col2, col3);
LOOKUP ON lookup_edge_1 WHERE lookup_edge_1.col1 >1 and lookup_edge_1.col1 < 3 and lookup_edge_1.col2 > 1;
Execution succeeded (Time spent: 2.461/4.992 ms)
```

Reply:
Got it